ChangeLog.md

```diff
@@ -46,6 +83,7 @@
 * Added the configuration files directory to the Help tab as it is located separately on Linux systems
 * Removed H3 data language selection during setup in favor of auto-detection
 * Replaced checkboxes with toggle buttons for easier of access on touchscreens.
+* Icons and translations now embedded in executable file
```

Comment:
Should this be here?

Reply:
Yeah. Missed change. It was part of 1.5.3. 

Noticed when similar change was made for 1.5.4 for map editor